README.md

```diff
@@ -1,88 +1,22 @@
-# Sylveon
+# Gradescope API
 
 [![CircleCI Status](https://dl.circleci.com/status-badge/img/circleci/3xWZeNMCu5Vj5AMLtYkBLD/3BbpjRhT2mTt2SAzHyziCj/tree/main.svg?style=svg&circle-token=a3755171dc7af90c9eed83c7eb96aa5f59faef63)](https://dl.circleci.com/status-badge/redirect/circleci/3xWZeNMCu5Vj5AMLtYkBLD/3BbpjRhT2mTt2SAzHyziCj/tree/main)
```

Comment:
Should probably comment or remove this for now since this is pointing to the template repo rather than this repo. For CI I'm leaning towards using GH actions also since it doesn't require a CircleCI account to view logs.

Reply:
Ah, I forgot to remove that, good point. I just removed it in the newest commit.